### PR TITLE
Debugging manuals

### PR DESCRIPTION
**Pull request by giobart**
Added debugging manuals covering:

- Control Plane observability
- Running Applications
- Networking
- VS Code Debuggin (draft mode activated, this manual release is deferred)

### CODE REVIEW

File: content/docs/manuals/debugging/control-plane.md [hunk 1]
```diff
@@ -1,12 +1,144 @@
...
+## How can I access Root and Cluster orchestrator logs?
+
+You have two ways: **(A)** Grafana Dashboard or **(B)** docker logs.
+
+### Using Grafana Dashboard
+
+The Grafana dashboards are exposed at <root_orchestrator_ip>:3000 and <cluster_orchestrator_ip>:3001, respectively. 
+
+{{< callout context="caution" title="Caution" icon="outline/alert-triangle" >}}
+The cluster Grafana dashboard is not available for 1-DOC deployments. For 1-DOC all the data is aggregated in the same dashboard.
```
Comment (nitindermohan) [inline content/docs/manuals/debugging/control-plane.md]: Are we still referencing 1-DOC? I thought that was overthrown with single machine and multiple machine deployment?

Please look into this and also add link to the setup doc

File: content/docs/manuals/debugging/control-plane.md [hunk 2]
```diff
@@ -9,4 +9,112 @@ seo:
...
+
+Or you can manually access the logs in 
+
+```
+/var/log/oakestra/nodeengine.log
+```
+
+### Net Manager
+
+On Net Manager v0.4.203 and above you can access the logs at
```
Comment (nitindermohan) [inline content/docs/manuals/debugging/control-plane.md]: Not sure if adding specific version in text is a good idea. Tie it down to the current version docs

File: content/docs/manuals/debugging/control-plane.md [hunk 3]
```diff
@@ -9,4 +9,112 @@ seo:
...
+
+On Net Manager v0.4.203 and above you can access the logs at 
+
+```
+/var/log/oakestra/netmanager.log
+```
+
+## How do I activate Debug Mode in NetManager? 
+
+In NetManager v0.4.302 or above add the debug `true` flag in your `netcfg.json` file as follows:
```
Comment (nitindermohan) [inline content/docs/manuals/debugging/control-plane.md]: Same issue with versioning here

File: content/docs/manuals/debugging/control-plane.md [hunk 4]
```diff
@@ -9,4 +9,112 @@ seo:
...
+  "NodePublicAddress": "0.0.0.0",
+  "NodePublicPort": "50103",
+  "ClusterUrl": "0.0.0.0",
+  "ClusterMqttPort": "10003",
+  "Debug": true
+}
+```
+3: Restart the Worker Node using `sudo NodeEngine start`
+
+{{< callout context="note" title="v0.4.301 or previous releases" icon="outline/info-circle" >}}
```
Comment (nitindermohan) [inline content/docs/manuals/debugging/control-plane.md]: Very version specific information. Maybe term it as "legacy"?

**Review by nitindermohan — CHANGES_REQUESTED** [review]
Please see inline comments, commits https://github.com/nitindermohan/oak-new-docs/commit/716a7df29318d22d5e3df01d196c7ae0fd814e11 and https://github.com/nitindermohan/oak-new-docs/commit/a9f2c67c52cad17cc874caf8bc8368c652633538 for changes

File: content/docs/manuals/debugging/control-plane.md (hunk 1 above)
Comment (giobart) [reply content/docs/manuals/debugging/control-plane.md]: true, I wrote this part before we changed naming. I'll fix this.

**Comment by giobart** [comment]
> Please see inline comments, commits [716a7df](https://github.com/nitindermohan/oak-new-docs/commit/716a7df29318d22d5e3df01d196c7ae0fd814e11) and [a9f2c67](https://github.com/nitindermohan/oak-new-docs/commit/a9f2c67c52cad17cc874caf8bc8368c652633538) for changes

@nitindermohan  Why in commit [716a7df](https://github.com/nitindermohan/oak-new-docs/commit/716a7df29318d22d5e3df01d196c7ae0fd814e11)  the entire cluster logs sections was deleted?

**Comment by nitindermohan** [comment]
@giobart Regarding https://github.com/nitindermohan/oak-new-docs/pull/17#issuecomment-2535505066,the deleted text was moved to the overview page: https://github.com/nitindermohan/oak-new-docs/blob/716a7df29318d22d5e3df01d196c7ae0fd814e11/content/docs/manuals/debugging/debugging-overview.md

File: content/docs/manuals/debugging/control-plane.md [hunk 5]
```diff
@@ -9,4 +9,109 @@ seo:
...
+  "NodePublicAddress": "0.0.0.0",
+  "NodePublicPort": "50103",
+  "ClusterUrl": "0.0.0.0",
+  "ClusterMqttPort": "10003",
+  "Debug": true
+}
+```
+3: Restart the Worker Node using `sudo NodeEngine start`
+
+{{< callout context="note" title="legacy v0.4.301 or previous releases" icon="outline/info-circle" >}}
```
Comment (nitindermohan) [inline content/docs/manuals/debugging/control-plane.md]: in title "legacy" -> "Legacy"

**Review by nitindermohan — APPROVED** [review]
minor change in lettering but approved otherwise

File: content/docs/manuals/debugging/control-plane.md (hunk 5 above)
Comment (giobart) [reply content/docs/manuals/debugging/control-plane.md]:
```suggestion
{{< callout context="note" title="Legacy v0.4.301 or previous releases" icon="outline/info-circle" >}}
```

**Comment by nitindermohan** [comment]
Merged. Thanks for all the work @giobart

**Pull request merged**